### DISCUSSION TITLE
Expand DiskDrive

### DISCUSSION THREAD

**Discussion by Sharkie49** [ubenmackin/ACVM#16] [General]
Hey!
First of all, thank you for working on this! I really like to see Windows on M1 Macs becoming reality!
So now to my problem/question:
I've managed to get everything installed and working but I am really searching for an possibility to expand the Disk Volume.
My attempt was to create a 100GB sized Disk with the "Create Disk" tool and mount it in the CD Image Window. I see the Drive now in my Disk Utility on Windows, but it's formated as RAW. 
So how can I access this "Virtual External Drive" to read and write or expand my main Volume?

Thank's in advance :)

**Comment by ubenmackin** [comment]
@Sharkie49 Sorry for the delay in response. Are you looking to expand the main drive in Windows? If so, take a look at these steps:

https://forums.macrumors.com/threads/success-virtualize-windows-10-for-arm-on-m1-with-alexander-grafs-qemu-hypervisor-patch.2272354/post-29384878

Here he is talking about converting the Windows VHDX image Microsoft provides to qcow2 format, and then expanding it in Windows.

Or are you asking if you can setup a second drive in Windows? If so, currently don't support a second hard drive (currently it will be mounted as a CD image).
